actors/miner/src/lib.rs

```diff
@@ -816,6 +816,8 @@ impl Actor {
             network_baseline: rew.this_epoch_baseline_power,
             circulating_supply,
             epoch_reward: rew.this_epoch_reward_smoothed,
+            epochs_since_ramp_start: pwr.ramp_start_epoch - rt.curr_epoch(),
```

Comment:
@lemmih - have updated the calls for the pledge function to use the state and subtract from current epoch like this.